airqo-platform/AirQo-frontend
# Ft branding segregation

- **What does this PR do?**
Users able to see the logo of their organization 

- **How do I test it out?**
cd `AirQo-frontend/netmanager`
```
npm install 

For Windows:
npm run stage-pc

For Mac:
npm run stage-mac
```
Try to the logo as KCCA or AirQo to see how the logo displayed varies
- Which JIRA card is associated?


- **Any other considerations?**
NONE

![Screenshot 2020-10-13 at 06 12 18](https://user-images.githubusercontent.com/15224992/95851434-e2e7ec80-0d5a-11eb-9531-0216707c2ba1.png)
![Screenshot 2020-10-13 at 06 11 35](https://user-images.githubusercontent.com/15224992/95851438-e54a4680-0d5a-11eb-8768-7c7413854bd7.png)
